wapitiCore/wappalyzer/wappalyzer.py

```diff
@@ -129,6 +140,25 @@ def normalize_application_regex(self):
 
                     for i, pattern in enumerate(self.applications[application_name][dict_field][key]):
                         self.applications[application_name][dict_field][key][i] = self.normalize_regex(pattern)
+            
+            # Format each dom of applications
+            tmp_dom = []
+            for (key, patterns) in self.applications[application_name]["dom"].items():
+                tmp_dom.append(self.normalize_application_regex_dom({key:patterns}))
+            self.applications[application_name]["dom"] = tmp_dom
+
+    def normalize_application_regex_dom(self, dom : dict) -> dict:
+        """
+        Convert the result of wappalyzer to a generic format used by Wapiti.
+        The goal is to match with the css selector if we don't have a regex.
+        In that case we set an empty value else we put the regex.
+        """
+        css_selector = list(dom.keys())[0]
+        value = dom[css_selector]
+        if value == "":
+            return {css_selector : {"exists": ""}}
+        else:
+            return dom        
```

Comment:
Here, what if dom contains multiple regular expressions? The 'first' could be empty but not the others

Reply:
There is only one. I do that to fit with the wapiti format which is list.